main_test.go

```diff
@@ -29,6 +25,8 @@ func wait(ch chan *sse.Event, duration time.Duration) (*sse.Event, error) {
 	return msg, err
 }
 
+/*
```

Comment:
Should this block be removed?

Reply:
need to redo all of these tests altogether. The integration tests cover this functionality too, so it should be ok to merge and address this later